lib/Elevate/Components/DatabaseUpgrade.pm

```diff
@@ -16,6 +15,20 @@ use Elevate::Database ();
 
 use parent qw{Elevate::Components::Base};
 
+use Cpanel::MysqlUtils::RemoteMySQL::ProfileManager ();
+
+use File::Slurper;
+use Try::Tiny;
+
+use Cpanel::MysqlUtils::MyCnf::Basic                ();
+use Cpanel::MysqlUtils::RemoteMySQL::ProfileManager ();
+use Cpanel::PasswdStrength::Generate                ();
+use Cpanel::JSON                                    ();
+use Cpanel::SafeRun::Simple                         ();
+use Cpanel::Encoder::URI                            ();
+
+use constant MYSQL_PROFILE_FILE => '/var/cpanel/elevate-mysql-profile';
```

Comment:
As far as I can tell, this isn't being created? At no point has Elevate tried to restore the remote instances after this whether they are still valid or not.